leaf-ui/js/onFunctions.js

```diff
@@ -753,6 +754,15 @@ paper.on("link:options", function (cell) {
                 selectResult.set('selected', false);
             }
 
+            var elements = SliderObj.getIntentionsAndActorsView();
```

Comment:
I'm wondering what are these codes for? I don't think it's being used in this method or I'm just missing something

Reply:
I think this was to make sure all elements show up in modeling mode, even if they had been hidden at the most recent time point due to their presence condition interval

Reply:
initial comment re: exclusion interval storage when increasing max abs time was fixed in commit 8d1a57e